crates/pathfinder/src/state/sync.rs

```diff
@@ -1162,8 +1158,10 @@ mod tests {
     async fn l2_noop(
         _: mpsc::Sender<l2::Event>,
         _: WebsocketSenders,
-        _: impl ClientApi,
-        _: Option<(StarknetBlockNumber, StarknetBlockHash, StateCommitment)>,
+        _: impl GatewayApi,
+        _: Option<(BlockNumber, BlockHash, StateCommitment)>,
+        // _: impl GatewayApi,
```

Comment:
rebase side-effects? :)

Reply:
I haven't force pushed my rebase yet but I do recall removing this while solving the rebase, so once I push it'll be fixed.